pyEfis.py

```diff
@@ -34,7 +34,6 @@
     from PyQt4.QtCore import *
 
 from decimal import *
-from aerocalc import std_atm
```

Comment:
Yes, but after I removed all the gui stuff from the main module this particular import wasn't needed anymore.  I think we still import it where needed.  My thinking on data like TAS, CAS, Density altitude and the like was to calculate it outside the graphical environment and put the data in the database like all the rest of the points.  If you look at the composite.py hook module you'll see that I calculated Average EGT, Max CHT and total Fuel in a separate routine.  Something similar could be done with TAS and then all the graphical parts could simply read TAS from the database.  That way if there is a device out on the network that is calculating TAS we can turn that hook off and the instrument doesn't have to change, it just still get's it from the database like normal.  I haven't gotten around to implementing it on air data yet, but that was my thinking.